anvil/src/udev.rs

```diff
@@ -323,6 +285,89 @@ pub fn run_udev(log: Logger) {
         state.device_added(&mut display, dev, path.into())
     }
 
+    if let Ok(instance) = Instance::new(Version::VERSION_1_2, None, log.clone()) {
```

Comment:
for purposes of testing, would it be useful to have a way to ignore vulkan and directly use gbm?